contracts/burner/Cargo.toml

```diff
@@ -36,6 +36,7 @@ cosmwasm-schema = { path = "../../packages/schema" }
 cosmwasm-std = { path = "../../packages/std", features = ["iterator", "cosmwasm_1_4"] }
 schemars = "0.8.3"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+enumset = "1.1.0"
```

Comment:
Does the warning happen because of cosmwasm-vm wasmer deps? Shouldn't this be a dev-dependency then?
Also: Can you add a comment why we need it, since we don't use it directly.

Reply:
This is an error in waster-types, about a missing / private trait:

```
    Checking wasmer-types v4.2.1
error[E0603]: trait `EnumSetTypeRepr` is private
  --> /Users/mauro/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasmer-types-4.2.1/src/initializers.rs:4:26
   |
4  | use enumset::__internal::EnumSetTypeRepr;
   |                          ^^^^^^^^^^^^^^^ private trait
   |
note: the trait `EnumSetTypeRepr` is defined here
  --> /Users/mauro/.cargo/registry/src/index.crates.io-6f17d22bba15001f/enumset-1.0.7/src/lib.rs:90:9
   |
90 |     use super::*;
   |         ^^^^^
```

This is just a workaround to force using a higher version of enumset.

Reply:
Will add a comment

Reply:
And you're right, this can be a dev-dependency.